src/map/battle.c

```diff
@@ -7436,6 +7436,7 @@ static const struct battle_data {
 	{ "critical_max_limit",                 &battle_config.critical_max,                    SHRT_MAX, 1,    INT_MAX,        },
 	{ "hit_min_limit",                      &battle_config.hit_min,                         1,      1,      INT_MAX,        },
 	{ "hit_max_limit",                      &battle_config.hit_max,                         SHRT_MAX, 1,    INT_MAX,        },
+	{ "autoloot_adjust",                    &battle_config.autoloot_adjust,                 SHRT_MAX, 1,    INT_MAX,        },
```

Comment:
look like wrong default and min/max values

Reply:
updated.